octodns/source/tinydns.py

```diff
@@ -21,13 +21,15 @@
 class TinyDnsBaseSource(BaseSource):
     SUPPORTS_GEO = False
     SUPPORTS_DYNAMIC = False
+    SUPPORTS_ROOT_NS = True
     SUPPORTS = set(('A', 'CNAME', 'MX', 'NS', 'TXT', 'AAAA'))
 
     split_re = re.compile(r':+')
 
     def __init__(self, id, default_ttl=3600):
         super(TinyDnsBaseSource, self).__init__(id)
         self.default_ttl = default_ttl
+        self.manage_root_ns = True
```

Comment:
Would prefer this be passed in as a parameter to super instead of setting the parent class's property here in the child.